### ISSUE TITLE
Grafbase deployment and project creation error

### ISSUE THREAD

**Issue by kt946** [adrianhajdin/project_nextjs13_flexibble#30]
Hello, I am following the video on Youtube up to the 3:00:00 mark and trying to deploy the grafbase project, but the deployment fails and I am getting this error "Environment variable NEXTAUTH_SECRET is not set" even though I created the root .env and the .env in the grafbase directory. Both of them have the NEXTAUTH_SECRET.

![image](https://github.com/adrianhajdin/project_nextjs13_flexibble/assets/103476893/0082b061-981a-4aea-a093-aa09a1ede3b4)

I have this in my grafbase.config.ts
![image](https://github.com/adrianhajdin/project_nextjs13_flexibble/assets/103476893/349b1245-a7c7-4972-8c9e-4b227eb56473)

I am also unable to create projects. When submitting projects I get this error:
```
Error: GraphQL Error (Code: 200): {"response":{"error":"{\"data\":null,\"errors\":[{\"message\":\"Unauthorized\"}]}","status":200,"headers":{"map":{"content-length":"51","content-type":"text/xml"}}},"request":{"query":"\n\tmutation CreateProject($input: ProjectCreateInput!) {\n\t\tprojectCreate(input: $input) {\n\t\t\tproject {\n\t\t\t\tid\n\t\t\t\ttitle\n\t\t\t\tdescription\n\t\t\t\tcreatedBy {\n\t\t\t\t\temail\n\t\t\t\t\tname\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t}\n","variables":{"input":{"title":"test","description":"test","image":"http://res.cloudinary.com/dkjbqkxbt/image/upload/v1692362385/jbvoynflj2tiufgfdnpr.png","liveSiteUrl":"https://github.com/adrianhajdin/project_nextjs13_flexibble","githubUrl":"https://github.com/adrianhajdin/project_nextjs13_flexibble","category":"Full-Stack","createdBy":{"link":"user_01H63XPMGTE1D8GVN3TDC5WZKN"}}}}}
    at makeRequest (webpack-internal:///(app-pages-browser)/./node_modules/graphql-request/build/esm/index.js:301:15)
    at async makeGraphQLRequest (webpack-internal:///(app-pages-browser)/./lib/actions.ts:23:16)
    at async handleFormSubmit (webpack-internal:///(app-pages-browser)/./components/ProjectForm.tsx:36:17)
```

Any assistance is greatly appreciated.

**Comment by fbjork** [comment]
@kt946 Hi there, you need to create the environment variable in the Grafbase dashboard too. The .env file is only used in local development.

**Comment by kt946** [comment]
@fbjork
Update: So I finally was able to fix the grafbase issue thanks to your comment! My Grafbase dashboard did not have the required environment variable in its settings. Not sure if I had to do it manually or it would automatically do it for me when connecting to the github repository, but in any case, the issue was fixed and grafbase deploys successfully. As for creating projects, I followed the video for creating the ProjectForm.tsx up to around the 3:00:00 mark, but to resolve my issue, I just copied the finished ProjectForm.tsx from adrianhajdin's repository and that resolved my creation issue. I can see the project data from the pathfinder. I probably had a small typo somewhere down the line, but at least with this, I can now proceed to finish the tutorial.
